src/app/navigation/root-stack.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { NavigatorScreenParams } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+import { MainAppNavigation, MainDrawerParamList } from './main-navigation';
+
+export type RootStackParamList = {
+  // TODO : onboarding , auth, etc
+  Main: NavigatorScreenParams<MainDrawerParamList>;
+};
```

Comment:
We should apply the same for this case.

Reply:
will remove this and put it inside a types/navigation.ts file